cmd/cloud/cluster.go

```diff
@@ -574,7 +574,7 @@ func defaultClustersTableData(clusters []*model.ClusterDTO) ([]string, [][]strin
 			provisionerMetadata.Version,
 			fmt.Sprintf("%d x %s", masterCount, masterInstanceType),
 			fmt.Sprintf("%d x %s (max %d)", provisionerMetadata.NodeMinCount, provisionerMetadata.NodeInstanceType, provisionerMetadata.NodeMaxCount),
-			provisionerMetadata.AMI,
+			NormalizeAMISuffix(provisionerMetadata.AMI),
```

Comment:
So instead of normalizing the AMI value in the cluster table, shouldn't we be storing the normalized value directly? That seems less confusing overall and eliminates the need for redoing it when showing cluster config.

Reply:
Not sure how this plays with your recent changes having AMIs per instance group. Let's discuss on planning to find a common decision

Reply:
Removed for now. If the representation is not ok for us let's improve it on another iteration. This part was totally minor for the purpose of this PR.